File: code-samples/python/consignment-order.py
import requests

url = "https://vendor-api-staging.jumia.com/consignment-order"

payload = {
    "shopId": "string",
    "businessClientCode": "string",
    "shippingDate": "string",
    "comment": "string",
    "products": [
        {
            "sku": "string",
            "quantity": 0,
            "labelCode": "string"
        }
    ]
}
headers = {"Authorization": "Bearer "}

response = requests.request("POST", url, json=payload, headers=headers)

print(response.text)

File: code-samples/go/consignment-order.go
package main

import (
  "fmt"
  "strings"
  "net/http"
  "io/ioutil"
)

func main() {

  url := "https://vendor-api-staging.jumia.com/consignment-order"

  payload := strings.NewReader("{\n  \"shopId\": \"string\",\n  \"businessClientCode\": \"string\",\n  \"shippingDate\": \"string\",\n  \"comment\": \"string\",\n  \"products\": [\n    {\n      \"sku\": \"string\",\n      \"quantity\": 0,\n      \"labelCode\": \"string\"\n    }\n  ]\n}")

  req, _ := http.NewRequest("POST", url, payload)

  req.Header.Add("Authorization", "Bearer ")

  res, _ := http.DefaultClient.Do(req)

  defer res.Body.Close()
  body, _ := ioutil.ReadAll(res.Body)

  fmt.Println(res)
  fmt.Println(string(body))

}